### PR TITLE
Added DCS Tests for System.Drawing Types

### LINKED ISSUE #25729: Add tests for DataContractSerializer serializing System.Drawing types
We need have tests covering DataContractSerializer serializing the following types.
Refer https://github.com/dotnet/corefx/blob/master/src/System.Runtime.Serialization.Xml/tests/DataContractSerializer.cs on how to add the DCS tests using the common helper method.
#20329

- [ ] System.Drawing.Point
- [ ] System.Drawing.PointF
- [ ] System.Drawing.Rectangle
- [ ] System.Drawing.RectangleF
- [ ] System.Drawing.Size
- [ ] System.Drawing.SizeF

### PR DESCRIPTION
**Pull request by clarkis117**
This adds DataContractSerializer tests for basic types in System.Drawing
	-See issue #25729 for a list of types

cc: @huanwu, @safern 

Resolves #25729

### CODE REVIEW

**Comment by stephentoub** [comment]
@huanwu, any concerns with this PR?

**Comment by huanwu** [comment]
@clarkis117 Could you put the tests in your project so you can maintain and catch any regression issues?

**Comment by clarkis117** [comment]
@huanwu I'm not sure what you are asking... Did you want the tests moved to the System.Drawing project?

File: src/System.Runtime.Serialization.Xml/tests/DataContractSerializer.cs [hunk 1]
```diff
@@ -2586,6 +2586,138 @@ public static void DCS_NonGenericDictionaryOfStringInt32()
         Assert.StrictEqual(true, Enumerable.SequenceEqual(value.Values.Cast<int>().ToArray(), deserialized.Values.Cast<int>().ToArray()));
     }
 
+    #endregion
+
+#region Drawing types
```
Comment (ahsonkhan) [inline src/System.Runtime.Serialization.Xml/tests/DataContractSerializer.cs]: nit: Maybe add a new test file with System.Drawing specific tests.

**Comment by stephentoub** [comment]
@huanwu, what's needed here? Thanks.

**Comment by huanwu** [comment]
@clarkis117 yes, could you move tests to the System.Drawing project?

**Comment by clarkis117** [comment]
@karelz Yes, I'm in the progress of making the changes. I just had to move a dependency, Utils.cs, for /Common/tests/System/Runtime/Serialization/DataContractSerializerHelper.cs from the System.Runtime.Serialization project into the  /Common/tests/System/Runtime/Serialization/ folder. Then update the projects that reference the file.

File: src/System.Runtime.Serialization.Xml/tests/System.Runtime.Serialization.Xml.Tests.csproj [hunk 2]
```diff
@@ -29,6 +28,7 @@
     <Compile Include="$(CommonTestPath)\System\Runtime\Serialization\DataContractSerializerHelper.cs">
       <Link>Common\System\Runtime\Serialization\DataContractSerializerHelper.cs</Link>
     </Compile>
+	<Compile Include="$(CommonTestPath)\System\Runtime\Serialization\Utils.cs" />
```
Comment (safern) [inline src/System.Runtime.Serialization.Xml/tests/System.Runtime.Serialization.Xml.Tests.csproj]: Nit: indentation.

**Comment by stephentoub** [comment]
Thanks!

**Pull request merged**